server/conn.go

```diff
@@ -1963,8 +1973,8 @@ func (cc *clientConn) prefetchPointPlanKeys(ctx context.Context, stmts []ast.Stm
 		}
 	}
 	pointPlans := make([]plannercore.Plan, len(stmts))
-	var idxKeys []kv.Key //nolint: prealloc
-	var rowKeys []kv.Key //nolint: prealloc
+	var idxKeys []kv.Key // nolint: prealloc
+	var rowKeys []kv.Key // nolint: prealloc
```

Comment:
delete space here, it will cause linter error.

Reply:
done